pkg/driver/agent.go

```diff
@@ -66,7 +66,7 @@ func NewNode(d *CSIDriver) csi.NodeServer {
 
 	// start the lvm node resource watcher
 	go func() {
-		err := lvmnode.Start(&ControllerMutex, stopCh)
+		err := lvmnode.Start(d.config.NodeControllerPollingInterval, &ControllerMutex, stopCh)
```

Comment:
Can you add the new argument as the last argument instead of the first?

Reply:
Fixed